lxa_iobus/server/server.py

```diff
@@ -432,7 +432,7 @@ async def set_pin(self, request):
             if value == "toggle":
                 await node.od.outputs.toggle(pin_name)
 
-            elif value:
+            elif int(value):
```

Comment:
Why not have the integer conversion above while creating the `value` variable? `value = int(post["value"])`

Reply:
That's what I did first, but "toggle" is also a valid `value` but not an integer.

Reply:
Ah, that's unfortunate...

Reply:
Yeah, the API is full of surprises here. I assumed that clients are expected to send  `Content-Type: application/json` here (which represents numbers as numbers), but in practice they send `Content-Type: application/x-www-form-urlencoded` (which represents everything as string).